tests/foreman/cli/test_leapp_client.py

```diff
@@ -382,3 +311,7 @@ def test_leapp_upgrade_rhel_non_admin(
     custom_leapp_host.power_control(state='reboot')
     result = module_target_sat.cli.JobInvocation.info({'id': invocation_command['id']})
     assert result['success'] == '1'
+
+    custom_leapp_host.clean_cached_properties()
```

Comment:
just curious, why its required to clean cached properties?

Reply:
It was there in the test earlier and I am guessing that it was added because we might be getting some cached version which was causing the issue.

Reply:
The `_os_release` is cached from before the upgrade. In order `_os_release` to return the correct OS version, we need to clean the cached value first.